scout/server/blueprints/cases/templates/cases/matchmaker.html

```diff
@@ -0,0 +1,236 @@
+      {{ case.display_name }}
+    </a>
+  </li>
+  <li class="active">
+    <span class="navbar-text">MatchMaker</span>
+  </li>
+{% endblock %}
+
+{% block content_main %}
+<div class="container">
+  <div class="panel panel-default">
+    <div class="panel-body">
+      <ol class="breadcrumb">
+        <li>Submitted: <strong>{{case.mme_submission.created_at.strftime('%Y-%m-%d %H:%M')}}</strong></li>
+        <li>Last updated: <strong>{{case.mme_submission.updated_at.strftime('%Y-%m-%d %H:%M')}}</strong></li>
+      </ol>
+      <div id="patientTab" class="container">
+        <ul class="nav nav-tabs">
+  			<li class="active">
+          <a  href="#1" data-toggle="tab">Patient Overview</a>
+  			</li>
+  			<li><a href="#2" data-toggle="tab">External Matches</a>
+  			</li>
+  			<li><a href="#3" data-toggle="tab">Internal Matches</a>
+  			</li>
+  		</ul>
+  			<div class="tab-content ">
+  			  <div class="tab-pane active" id="1">
+            {{patient_data()}}
+  				</div>
+  				<div class="tab-pane" id="2">
+            {{show_matches('external')}}
+  				</div>
+          <div class="tab-pane" id="3">
+            {{show_matches('internal')}}
+  				</div>
+  			</div>
+      </div>
+    </div>
+  </div>
+</div>
+{% endblock %}
+
+{% macro patient_data() %}
+<hr></hr>
+<div>
+  <p>Features (HPO terms):
+    {% for hpo in case.mme_submission.features %}
+      <span class="label label-primary">{{hpo.id}}</span>
+    {% endfor %}
+  </p>
+  <p>Disorders (OMIM terms):
+    {% for omim in case.mme_submission.disorders %}
+      <span class="label label-info">{{omim.id}}</span>
+    {% endfor %}
+  </p>
+  <p>Submitted patients:
+    {% for patient in case.mme_submission.patients %}
+      <ul>
+        <li>Patient id: <strong>{{patient.id}}</strong></li>
+        <li>Patient label: <strong>{{patient.label}}</strong></li>
+        <li>Sex: <strong>{{patient.sex}}</strong></li>
+        <li>genomic features:
+          <div class="container-fluid">
+            <div class="row">
+            {% for g_feat in patient.genomicFeatures %}
+              <div class="col-lg-4">
+                <div class="panel panel-default">
+                  <div class="panel-body">
+                    {% for key, value in g_feat.items() %}
+                      {% if key == 'gene' %}
+                        {{key}}:&nbsp;<span class="label label-default">{{value.id}}</span>
+                        <hr>
+                      {% elif key == 'variant' %}
+                        {% for ikey, item in value.items() %}
+                          {{ikey}}:<strong>{{item}}</strong><br>
+                        {% endfor %}
+                      {% elif key == 'type' %} <!-- this will be variant effect -->
+                        {{key}}:<strong>{{value.label}}</strong>
+                        <br>
+                      {% else %} <!-- this will be zygosity -->
+                        {{key}}:<strong>{{value}}</strong>
```

Comment:
Is this way of describing zygosity a matchMaker standard? Otherwise I would suggest casting this as a string het / hom / hemi. I'm sure our users can learn, but this at least needs documentation. Also, flagging hemis (at least when homozygous state on X for a male) will be very useful, but could wait until a later release if it is complicated with the current setup!

Reply:
The standard is defined in this document: https://github.com/ga4gh/mme-apis/blob/master/search-api.md

It says: 
zygosity: <number> (1 for heterozygous or hemizygous, 2 for homozygous)

You can't really distinguish between heteroz and hemiz because they both have 1. Didn't know how to render it from out data..

Reply:
Slightly unusual - we usually clump hom with hemi, but confirmed: as long as it is in their standard, lets keep it. If you are feeling *very* user friendly, add a string tag next to it (1 is "hemi" for variants on X for males, "het" otherwise, and 2 can always get a "hom"). 👍

Reply:
Good idea, you are right, it's not obvious! Fixed!